### ISSUE TITLE
Mugwump Hardmantle (item)

### ISSUE THREAD

**Issue by Tesq** [WarEmu/WarBugs#19431]
These shoulder came as reward from last chain step of "Stunties For Da Forge ", TM chap 17 normal quest.
But it's an appereance of BG armor. For what i understand this quest should had some alternative appereance shoulder for marauders, namely 1 double appereance shoulder like chosen (as it does for some other classes) . The shoulders arent either bad but they are simply wrong positioned (too high) when char wear em (or use as appereance). if these cant be fix to be wear properly the appereance should/could be replaced with another double shoulder for mara or for mara but taken from some low level chosen shoulders.

![Draznicht_004](https://user-images.githubusercontent.com/7527415/132993501-2bf12a8e-dd08-47f7-b228-acd7e5c0aaea.jpg)

**Comment by lilimx** [comment]
tl;dr: wrong appearance

**Comment by lilimx** [comment]
Fixed with Patch 01/10/2021
